src/test/libical-glib/array.py

```diff
@@ -16,7 +16,7 @@
 
 import gi
 
-gi.require_version('ICalGLib', '3.0')
+gi.require_version('ICalGLib', '4.0')
```

Comment:
Will it make sense to also use `LIBICAL_LIB_MAJOR_SOVERSION` for this dependency?

Reply:
only if the test files are "configured" as the config.h or similar files, which is not feasible considering it is a test file, not a build file.

Reply:
OK. I see

Reply:
> Will it make sense to also use `LIBICAL_LIB_MAJOR_SOVERSION` for this dependency?

I thought about that.
It would be nice not to change all the test `.py` files by-hand every time we update the major version.
then we need to generate them at cmake stage, which should be possible.
in a separate patch.

Reply:
You do not change the version that often to it be any concern, no? The tests will fail soon when the project/soname version is changed, but the python tests not updated.

Reply:
> You do not change the version that often to it be any concern, no? The tests will fail soon when the project/soname version is changed, but the python tests not updated.

yes, that's true.
it would be "busy work" if someone is interested.